inc/saibuffer.h

```diff
@@ -262,6 +262,48 @@ typedef enum _sai_buffer_pool_threshold_mode_t
 
 } sai_buffer_pool_threshold_mode_t;
 
+/**
+ * @brief Enum defining buffer pool family attributes.
+ */
+typedef enum _sai_buffer_pool_family_attr_t
+{
+    /**
+     * @brief Start of attributes
+     */
+    SAI_BUFFER_POOL_FAMILY_ATTR_START,
+
+    /**
+     * @brief Shared buffer size in bytes
+     *
+     * This is derived from subtracting all reversed buffers of queue/port
```

Comment:
may be reserved buffers ?

Reply:
https://github.com/opencomputeproject/SAI/pull/1540/commits/91aef8168318af88f0953d22d8e484eb5afd4a45